src/index.ts

```diff
@@ -146,9 +165,7 @@ class Monika extends Command {
         const startupMessage = this.buildStartupMessage(config, flags.verbose)
 
         this.log(startupMessage)
-
-        // Loop through all probes
-        abortCurrentLooper = loopProbes(config)
+        idFeeder(config, Number(flags.repeat), flags.id)
```

Comment:
when abortCurrentLooper is removed, monika will not be able to update it's configuration if new configuration is received from Symon